acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
Also puppet defaults the pidfile based on run_mode, which is horrible because it requires that run-mode be set before settings are initialized:

<pre>
    :default  => "$rundir/${run_mode}.pid",
</pre>